components/locid_transform/src/fallback/mod.rs

```diff
@@ -409,7 +409,7 @@ impl LocaleFallbackerBorrowed<'static> {
 impl<'a> LocaleFallbackerWithConfig<'a> {
     /// Creates an iterator based on a [`DataLocale`].
     ///
-    /// If you have a [`Locale`], call `.into()` to get a [`DataLocale`].
+    /// If you have a [`Locale`](icu_locid::Locale), call `.into()` to get a [`DataLocale`].
```

Comment:
I think in docs we generally use the meta crate, so this should link to `icu::locid::Locale`

Reply:
We use `icu::` in docs tests but in links we always go to `icu_`, which is usually implicit because the type is being imported at the top of the file. We can't link to `icu::` because links only work in real deps not dev deps.